### DISCUSSION TITLE
2022-29: 在疫情期间得感冒

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#369] [Announcements]
# 2022-29: 在疫情期间得感冒

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-29/

**Comment by wfxr** [comment]
几年也前经历过一次，才明白抗生素对细菌引起的感冒效果很好，但对病毒没有任何作用。病毒性感冒如果没有对应的特效疫苗，基本上只能硬扛，所以医院不让打针或者打点滴可能是正确的。祝早点好起来～

**Comment by rickyyyxu** [comment]
经常清理空调滤网也是很重要的~

**Comment by lilydjwg** [comment]
病毒性感冒就是这样子的，只能熬着，保证水分和能量充足，好好休息，烧太高了才会用药物强行降点儿，也不是直接对付病毒的。
好在我最近几次感冒都没有发烧，休息着等病程过了就好了。

小时候我总觉得现代医学好发达啊，后来长大了才发现，现代医学能治好的病其实非常有限，病了之后大部分时候只能打辅助，或者绕过（比如切掉坏了的部分）。偶尔治愈，常常帮助，总是安慰。

**Reply by Xuanwo** [reply]
> 后来长大了才发现，现代医学能治好的病其实非常有限，病了之后大部分时候只能打辅助，或者绕过（比如切掉坏了的部分）

确实是这个体会
